### DISCUSSION TITLE
Getting Forecast Data

### DISCUSSION THREAD

**Discussion by zyberzebra** [meteostat/meteostat-python#38] [Ideas] [answered]
Will there be a way to access the forecast data via the python library or the general api?

Or will you just make the historical data available.

thx in advance

**Comment by clampr** [comment, marked as answer]
Actually, you can already access forecast data using Meteostat. Predictions are usually available for up to 10 days. Example: https://meteostat.net/en/station/71624?t=2021-04-29/2021-05-05

When using the JSON API, you'll need to opt-in for model data. Through Meteostat Python you should automatically be able to access forecast data when using the `Hourly` data interface.

**Reply by trangdo42** [reply]
Hi! Even though the webapp platform shows the forecasts for specific stations up to 10 days as you say, the Hourly data interface currently returns forecast only for the next day. For some, not even that and it returns only the historical data (although according to the webapp they're certainly not missing). Was there an update of the Meteostat Python I missed or is it from now on unavailable through this service? Thank you in advance!

**Reply by p4arth** [reply]
Hi! were you able to find any solution to this? Actually I am facing a similar issue too.

**Reply by clampr** [reply]
Can you share your script?

**Reply by jlmedinamba** [reply]
@clampr   - love meteostat. Quick question... can't discern in the docs what the actual weather forecast source is - so for the CONUS - it's unclear to me since NOAA provides so many forecast options (i.e. GFS, NAM, NBM etc.)

Can someone please clarify? Thx.

**Reply by clampr** [reply]
We use a mix of [MOSMIX](https://www.dwd.de/EN/ourservices/met_application_mosmix/met_application_mosmix.html;jsessionid=DA8E1BA38C4E7EB0DAF63653CCAF771F.live21064) and [Met.no](https://api.met.no/weatherapi/locationforecast/2.0/documentation).

**Reply by alvinhenrick** [reply]
@clampr Thank you for this wonderful API. Quick clarification? Is the hourly weather actuals values or historic forecasts values you captured and archived?
